scripts/train_speculator.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# On AWS, the EFA and OFI paths enable NCCL to use optimized networking.
+export LD_LIBRARY_PATH=/opt/nccl/build/lib:/opt/amazon/efa/lib:/opt/amazon/openmpi/lib:/opt/aws-ofi-nccl/lib:/usr/local/cuda/lib:/usr/local/cuda/lib64:/usr/local/cuda:/usr/local/cuda/targets/x86_64-linux/lib/:/usr/local/cuda/extras/CUPTI/lib64:/usr/local/lib:$LD_LIBRARY_PATH
+
+export FI_EFA_SET_CUDA_SYNC_MEMOPS=0
+
+MODEL_ARGS="\
```

Comment:
We may want to have a readme for the speculator parameters specifically so a user knows what to explicitly change in this script